spec/models/historical_event_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe HistoricalEvent, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I fixed the factories, I'm not sure what to do about this one. All of these were auto-generated. Should `pending...` be aligned with `RSpec.describe...`?